uniffi_bindgen/src/bindings/python/gen_python.rs

```diff
@@ -156,7 +156,7 @@ mod filters {
                 coerce_py(&"v", t)?,
                 nm
             ),
-            Type::Custom { .. } => panic!("No support for coercing external types yet"),
+            Type::Custom { primitive, .. } => coerce_py(nm, primitive.as_ref())?,
```

Comment:
I ended up with 2 new types - `External` and `Wrapped` and dropped the new "Kind" enum, which helped keep the templates fairly simple.